air/src/execution_step/instructions/ap/apply_to_arguments.rs

```diff
@@ -46,12 +49,24 @@ pub(super) fn apply_to_arg(
     Ok(result)
 }
 
-fn apply_const(value: impl Into<JValue>, exec_ctx: &ExecutionCtx<'_>, trace_ctx: &TraceHandler) -> ValueAggregate {
+fn apply_const(
+    value: impl Into<JValue>,
+    exec_ctx: &ExecutionCtx<'_>,
+    trace_ctx: &TraceHandler,
+) -> ExecutionResult<ValueAggregate> {
     let value = Rc::new(value.into());
     let tetraplet = SecurityTetraplet::literal_tetraplet(exec_ctx.run_parameters.init_peer_id.as_ref());
     let tetraplet = Rc::new(tetraplet);
 
-    ValueAggregate::new(value, tetraplet, trace_ctx.trace_pos())
+    Ok(ValueAggregate::new(
+        value,
+        tetraplet,
+        trace_ctx.trace_pos().map_err(|e| {
```

Comment:
IMHO, worth extracting to separate variable.

Reply:
Done